resume-sass/resume.html

```diff
@@ -0,0 +1,202 @@
+<!DOCTYPE html5>
+<html>
+	<head>
+		<title>Aya's Resume</title>
+		<style>
+			/*body{
```

Comment:
Try to never check-in commented code. They are bad for readability and maintainability.